pom.xml

```diff
@@ -81,7 +80,7 @@
         <pluginManagement>
             <plugins>
                 <plugin>
-                    <groupId>com.coveo</groupId>
+                    <groupId>com.spotify.fmt</groupId>
                     <artifactId>fmt-maven-plugin</artifactId>
                     <version>${fmt-maven-plugin.version}</version>
                     <configuration>
```

Comment:
Can we remove `skipSortingImports` now?

Reply:
I was about to do it, but then realized we will need to undo the imports in the autogenerated files every time, unless we find a way to exclude them somehow.

Reply:
The autogenerated files are ignored because of the config in the module though, yeah?

Reply:
I was hoping to find a way to utilize the `filesNamePattern` config option so that we apply it on everything except them.

Reply:
Gotcha, yeah we can look into that for a future improvement